.github/workflows/ci-windows.yml

```diff
@@ -0,0 +1,37 @@
+name: ci-windows
+
+on: [ push, pull_request ]
+
+jobs:
+  build:
+    runs-on: windows-latest
+    strategy:
+      matrix:
+        python-version: [ "3.10", "3.11", "3.12" ]
```

Comment:
Do you think it's worth / possible to add 3.9.2? That's the earliest version we're supporting

Reply:
Yes, of course. I was convinced we start version 3.10